docs/writing-tests/snapshot-testing.md

```diff
@@ -8,9 +8,56 @@ Storybook is a helpful tool for snapshot testing because every story is essentia
 
 ![Example Snapshot test](./snapshot-test.png)
 
-## Setup Storyshots
+## Set up test runner
 
-[Storyshots](https://storybook.js.org/addons/@storybook/addon-storyshots/) is the official Storybook addon that enables snapshot testing, powered by [Jest](https://jestjs.io/docs/getting-started).
+Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+
+Follow the [setup instructions](./test-runner#setup) to install and run the test runner.
+
+Then, you can use the [test hook API](./test-runner#test-hook-api-experimental) to snapshot the HTML of each story:
+
+```js
+// .storybook/test-runner.js
+
+module.exports = {
+  async postRender(page, context) {
+    // the #storybook-root element wraps each story
+    const elementHandler = await page.$('#storybook-root');
+    const innerHTML = await elementHandler.innerHTML();
+    expect(innerHTML).toMatchSnapshot();
+  },
+};
+```
```

Comment:
@kylegach a minor things for you to look into if you have a chance. Can put this into a snippet file and provide a TS variation? We have it [here](https://github.com/storybookjs/test-runner/blob/next/.storybook/test-runner.ts)